src/rendering/RenderUtil.cc

```diff
@@ -439,12 +439,17 @@ void RenderUtil::Update()
       // Don't move entity being manipulated (last selected)
       // TODO(anyone) Check top level visual instead of parent
       auto vis = std::dynamic_pointer_cast<rendering::Visual>(node);
+      int updateNode = 0;
       Entity entityId = kNullEntity;
       if (vis)
+      {
+        updateNode = std::get<int>(vis->UserData("pause-update"));
```

Comment:
can you add some comments on what `pause-update` is for and how it's used?

Reply:
Resolved in [afe5d75b](https://github.com/ignitionrobotics/ign-gazebo/pull/59/commits/afe5d75b8f2ee1b8b38247903c54b5d596f02760)